charts/pvault-server/Chart.yaml

```diff
@@ -10,6 +10,7 @@ dependencies:
     repository: https://charts.bitnami.com/bitnami
     condition: postgresql.enabled
 home: https://piiano.com/pii-data-privacy-vault/
-keywords: ["piiano", "vault", "pvault", "privacy", "tokenization", "encryption", "pci"]
+keywords:
```

Comment:
is it OK it moved a line?

Reply:
Changed by the auto formatter. idk. reverted